packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -121,10 +121,7 @@
     
     Detect these situations and skip updates to @(SupportedTargetFramework) etc. 
   --> 
-  <ItemGroup Condition="('$(UseWPF)' == 'true' Or '$(UseWindowsForms)' == 'true') And 
-                        '$(_TargetFrameworkVersionValue)' != '' And
-                        '$(_TargetFrameworkVersionValue)' != '$(_UndefinedTargetFrameworkVersion)' And 
-                        '$(_TargetFrameworkVersionValue)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)'">
+  <ItemGroup Condition=" '$(_EnableWindowsDesktopSupportUnsupportedTargetFramework)' == 'true' ">
```

Comment:
I think renaming this to something like `_RemoveSupportedTargetFrameworksForWindowsDesktop` would be clearer.

Reply:
Isn't the *intent* here better described as `_RemoveUnsupportedTargetFrameworksForWindowsDesktop` ?

Reply:
Where is `_EnableWindowsDesktopSupportUnsupportedTargetFramework` being set/calcualted to `true` ?

Reply:
`_RemoveUnsupportedTargetFrameworksForWindowsDesktop` sounds more accurate

Reply:
> Where is _EnableWindowsDesktopSupportUnsupportedTargetFramework being set/calcualted to true ?

here https://github.com/dotnet/wpf/blob/7baee74daa23d1e8933a35085821934b0788246d/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets#L73

Reply:
Every time we use this trick, we should document what we are doing 😄.

Reply:
renamed to _RemoveUnsupportedTargetFrameworksForWindowsDesktop